scripts/pr_reviewer.py

```
@@ -1,104 +1,230 @@
+                    continue  # skip script itself
+                rule = entry.get("rule", "")
+                file_issues.setdefault(path, []).append(rule)
+        return file_issues
+    except Exception as e:
+        return {"Gitleaks scan error": [str(e)]}
+
+
+# ---------------- COMPLIANCE SCAN ---------------- #
+def run_compliance_scan(repo_path="."):
+    file_issues = {}
+    for root, dirs, files in os.walk(repo_path):
```

Ruff (0.12.2)

Loop control variable `dirs` not used within loop body

Rename unused `dirs` to `_dirs`

(B007)